doc/swagger-api-description/api.yaml

```diff
@@ -445,36 +589,153 @@ definitions:
     properties:
       uuid:
         type: string
+        description: Virtual file UUID.
       file-properties:
         type: array
         items:
-          $ref: "#/definitions/FileProperty"
+          $ref: "#/definitions/Property"
       file-parts:
         type: array
         items:
           $ref: "#/definitions/PartLocation"
+      VirtualChildSet:
+        type: array
+        items:
+          $ref: "#/definitions/VirtualFile"
+      allocateModel:
+        $ref: "#/definitions/AllocationModel"
 
-  FileProperty:
+  Property:
     description: Describes all file property.
     type: object
     properties:
+      uuid:
+        type: string
+        description: Property UUID.
       property-name:
         type: string
+        description: Property name.
       property-value:
         type: string
+        description: Property value.
 
   PartLocation:
-    description: Describes the sequence number of file part and its location.
+    description: Describes sequence number of file part and its location.
     type: object
     properties:
       uuid:
         type: string
+        description: PartLocation UUID.
       number:
         type: integer
         format: int32
+        description: Sequence number.
       location:
         $ref: "#/definitions/Cloud"
 
+  AllocationModel:
+    description: Describes allocate storage model ot file.
+    type: object
+    properties:
+      uuid:
+        type: string
+        description: Allocation model UUID.
+      modelType:
+        description: Types of allocation model.
+        enum:
+          - UNION
+          - MORROR
+          - HIGH_AVAILABILITY
+      propertySet:
+        type: array
+        items:
+          $ref: "#/definitions/Property"
+
+  Statistics:
+    type: object
+    description: Contain statistic information about cloud.
+    properties:
+      uuid:
+        type: string
+        description: Statistic UUID.
+      cloudSet:
+        $ref: "#/definitions/Cloud"
+      uploadFiles:
+        description: Count of uploaded files.
+        type: integer
+        format: int32
+      uploadTime:
+        description: Time what files was uploaded.
+        type: integer
+        format: int64
+      downloadFiles:
+        description: Count of downloaded files.
+        type: integer
+        format: int32
+      downloadTimes:
+        description: Time what files was downloaded.
+        type: integer
+        format: int64
+
+  HelthStatistis:
```

Comment:
As I have written above: very abstract object. Please explain what it is going to be used for. 
_In a wisper_ : please correct spelling (he _a_ lth, spa _c_ e). Sorry for this.